src/H5queue.h

```diff
@@ -31,8 +31,8 @@
  *	@(#)queue.h	8.5 (Berkeley) 8/20/94
  */
 
-#ifndef	_SYS_QUEUE_H_
-#define	_SYS_QUEUE_H_
+#ifndef	_H5queue_H_
+#define	_H5queue_H_
```

Comment:
This should be `H5queue_H`. Otherwise you'll add more `Wreserved-id-macro` warnings.

Reply:
Ah. Will fix.

Reply:
I fixed this one as well as the other VFD SWMR include guards.